pkg/backup/volume_snapshot_action.go

```diff
@@ -38,11 +39,15 @@ type volumeSnapshotAction struct {
 
 var _ Action = &volumeSnapshotAction{}
 
-func NewVolumeSnapshotAction(snapshotService cloudprovider.SnapshotService) Action {
+func NewVolumeSnapshotAction(snapshotService cloudprovider.SnapshotService) (Action, error) {
```

Comment:
The more I think about it, I'm not sure that changing the signature here really provides much value?